src/functions/user/memberAvatar.js

```diff
@@ -1,21 +1,21 @@
 module.exports = async d => {
     const data = d.util.aoiFunc(d);
 
-    const [guildID = d.guild?.id, userID = d.author?.id, size = 4096, dynamic = 'true', extension] = data.inside.splits;
+    const [guildID = d.guild?.id, userID = d.author?.id, size = 4096, dynamic = "true", extension] = data.inside.splits;
 
     const guild = await d.util.getGuild(d, guildID);
-    if (!guild) return d.aoiError.fnError(d, 'guild', {inside: data.inside});
+    if (!guild) return d.aoiError.fnError(d, 'guild', {inside: data.inside})
 
-    const member = await d.util.getMember(guild, userID);
-    if (!member) return d.aoiError.fnError(d, 'member', {inside: data.inside});
+    const user = userID === d.author?.id ? d.member : await d.util.getMember(guild, userID);
+    if (!user) return d.aoiError.fnError(d, "member", {inside: data.inside});
 
-    data.result = member.avatarURL({
+    data.result = user.displayAvatarURL({
```

Comment:
well keeping it `avatarURL` should be fine as it also served as a way to check if member avatar exists or not 
so if u want to change it to `displayAvatarURL` can you also add a function to check if member avatar exists or not